pogam/scrapers/seloger.py

```diff
@@ -41,7 +43,20 @@
     )""",
     re.UNICODE | re.VERBOSE,
 )
-
+def map_cp_to_ci (cp: str) -> str:
+    """ convert CP to CI as seloger use CI by default for search."""
+    df = pd.read_csv('https://www.data.gouv.fr/en/datasets/r/6d3428b2-3893-45a1-b404-2522a4e77d41',sep=';')
+    cp=str(cp)    
+    try:
+        n=df[df['Code Postal'] == cp]['Code INSEE'].values[0]
+    except Indexerror:
+        msg = f"Unknown post code f{cp}"
+        raise ValueError(msg)
```

Comment:
`raise` will stop your program, so you will never reach the logger line.
you don't really need to log anything here, since you will see `msg` as an error msg. But if for some reason you wanted to, you should put it before the `raise`